api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -130,6 +130,10 @@ message RateLimit {
   // Sets the HTTP status that is returned to the client when the ratelimit server returns an error
   // or cannot be reached. The default status is 500.
   type.v3.HttpStatus status_on_error = 12;
+
+  // Optional additional prefix to use when emitting statistics. This allows to distinguish
+  // emitted statistics between configured ``ratelimit`` filters in an HTTP filter chain.
+  string stat_prefix = 13;
```

Comment:
Should update ratelimit stats docs as well: https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/rate_limit_filter.html#statistics

Reply:
done, ptal again, thanks.